src/main/java/frc5124/robot2020/subsystems/PanelController.java

```diff
@@ -24,148 +24,148 @@
 
 public class PanelController implements Subsystem {
```

Comment:
I'm going to assume we're just not ready for this and it is causing problems. otherwise we should not commit wide spread commented out code to master

Reply:
didn't want to delete so we wouldn't lose it. Unsure state of it. Might move to another branch